syntax = "proto3";

package db_subscription;

message db_data {
    int32 value1 = 1;
    string value2 = 2;
}